createNewJobFolders.py

```diff
@@ -55,19 +57,21 @@ def get_next_job_and_part_number():
     part_number = f"{max_part_number + 1:02d}"
     return job_number, part_number
 
-# Note the location of the yaml file
-script_dir = os.path.dirname(os.path.abspath(__file__))
-yaml_file_path = os.path.join(script_dir, 'config.yaml')
 
-# Open and read the yaml file
-with open(yaml_file_path, 'r') as f:
-    folder_structure = yaml.load(f, Loader=yaml.FullLoader)
+if __name__ == '__main__':
```

Comment:
This is just explicit marking to state this is the entrypoint of the applciation.